src/main/java/com/nike/cerberus/auth/connector/onelogin/OneLoginAuthConnector.java

```diff
@@ -157,7 +157,7 @@ protected SessionLoginTokenData verifyFactor(final String deviceId,
             String msg = String.format("stateToken: %s failed to verify 2nd factor for reason: %s",
                     stateToken, verifyFactorResponse.getStatus().getMessage());
 
-            if (verifyFactorResponse.getStatus().getCode() == 400L) {
+            if (verifyFactorResponse.getStatus().getCode() == 401) {
```

Comment:
You are introducing a bug here.

This code is verifying the response code from the OneLogin API call, which would still be a `400`, unless I am mistaken.